driver-core/src/test/unit/com/mongodb/internal/connection/DescriptionHelperSpecification.groovy

```diff
@@ -94,9 +94,9 @@ class DescriptionHelperSpecification extends Specification {
                                           maxMessageSizeBytes : 48000000,
                                           maxWriteBatchSize : 1000,
                                           localTime : ISODate("2015-03-04T23:03:45.848Z"),
-                                          maxWireVersion : 3,
+                                          maxWireVersion : 6,
                                           minWireVersion : 0,
-                                          compressors : ["zlib", "snappy"],
+                                          compression : ["zlib", "snappy"],
```

Comment:
And this isn't even right.  Discovered when adding the `==` below.  I double-checked and the server does reply with compression not compressors

Reply:
:man_facepalming: